target/linux/ath79/image/generic.mk

```diff
@@ -1119,6 +1123,32 @@ define Device/teltonika_rut955
 endef
 TARGET_DEVICES += teltonika_rut955
 
+define Device/teltonika_rut230
+  SOC := ar9330
+  DEVICE_VENDOR := Teltonika
+  DEVICE_MODEL := RUT230
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb-chipidea2 kmod-usb-net \
```

Comment:
kmod-usb-core and kmod-usb-serial can be removed here, as they will be selected by the other packages.

Reply:
kmod-usb-net can be removed as well.
Despite, wrap lines to not exceed 80 characters.